slave/process-fs-home/changelog

```diff
@@ -1,3 +1,10 @@
+perun-slave-process-fs-home (3.1.4) stable; urgency=medium
+
+  * Use temporary directory to prepare new home, set correct permission and
+    ownership and then move it to the right place safely
```

Comment:
s/safely/atomically/

Also it would be nice to explain we do this to overcome problems on distributed filesystems.

Reply:
Fixed and added.